### DISCUSSION TITLE
Einbinden JSON in Opendtu on battery

### DISCUSSION THREAD

**Discussion by firehawk1979** [arendst/Tasmota#22407] [Q&A]
Hallo liebe Community,
ich habe das ich den JSON String von meinem Bitshake Smartmeter (ESP32-C3) nicht in meine DTU eingebunden bekomme. Der String lässt sich über http auslesen, aber in der DTU bekomme ich nur eine Fehlermeldung.

String:

{"StatusSNS":{"Time":"2024-11-01T12:38:04","":{"Power_in":19548.4,"Power_out":0.0,"Power":537,"Meter_Number":"0a014c475a00037ea8e1"}}}

![Unbenannt](https://github.com/user-attachments/assets/aabcd267-610b-4182-a25d-fb79bbd46264)

**Comment by LinuxUnix80** [comment]
Moin,

habe auch Bitshaker. Und bei mir gehts. Bin da kein Profi aber bei dir ist nach Time  ein "" . Bei mir steht dort der Zählertyp (SGM) und damit gehts.

Beispiel Script auf Bitshaker Zeile 5 (Werkzeuge/Script):
+1,5,s,16,9600,SGM,4

String bei Opendtu on Battery:
StatusSNS/SGM/Power

String Bitshaker Rohdaten:
{"StatusSNS":{"Time":"2024-11-26T20:58:47","SGM":{"E_in":684.391,"E_out":12.799,"Power":935,"Voltage":223.2,"Voltage_L2":222.7,"Voltage_L3":223.6,"Power_L1_curr":106,"Power_L2_curr":571,"Power_L3_curr":257,"Current":0.96,"Current_L2":2.97,"Current_L3":1.69,"Freq":49.9,"ID":"


Wie sieht die Webabfrage aus (Browser> http://192.168.178.154/cm?cmnd=status%208) ?

Bei mir:

![grafik](https://github.com/user-attachments/assets/6e0afc99-0ec2-4a9f-9a30-09ccf779eefe)
